src/output/geojson.cpp

```diff
@@ -27,9 +25,17 @@ bool is_oneway(const vb::DirectedEdge *e) {
 namespace osmlr {
 namespace output {
 
-geojson::geojson(vb::GraphReader &reader, std::string base_dir, size_t max_fds)
-  : m_reader(reader)
+geojson::geojson(vb::GraphReader &reader, std::string base_dir, size_t max_fds,
+                 time_t creation_date, const uint64_t osm_changeset_id)
+  : m_osm_changeset_id(osm_changeset_id)
+  , m_reader(reader)
   , m_writer(base_dir, "json", max_fds) {
+  // Change cration date into string plus int
+  m_creation_date = creation_date;
+  std::tm tm = *std::gmtime(&creation_date);
+  char mbstr[100];
+  std::strftime(mbstr, sizeof(mbstr), "%c %Z", &tm);
```

Comment:
before you do this you need something like: `std::locale::global(std::locale("C"));` to make sure to avoid the users default locale